spec/models/storage_location_spec.rb

```diff
@@ -22,6 +22,26 @@
     end
   end
 
+  context "Callbacks >" do
+    describe "before_destroy" do
+      let!(:item) { create(:item) }
+      subject { create(:storage_location, :with_items, item_quantity: 10, item: item, organization: @organization) }
+
+      it "does not delete storage locations with inventory items on it" do
+        subject.destroy
+
+        expect(subject.errors.messages[:base]).to include("Cannot delete storage location containing inventory items")
+      end
+
+      it "deletes storage locations with no inventory items on it" do
+        subject.inventory_items.destroy_all
+        subject.destroy
```

Comment:
```suggestion
        result = subject.destroy
        expect(result).to eq(false)
```
Or check that the value returned is false since the controller depends on this to return false.

Reply:
should we expect false on this one? this is the case where we want the model to be deleted

Reply:
Yeah I think you want `true` here since it's happy path; could use the `false` logic in the previous example though, where you're testing an unsuccessful `:destroy`?

Reply:
Ahh right it should be `true`!